powershell/public/maester/exchange/Test-MtExoMoeraMailActivity.ps1

```diff
@@ -44,8 +44,8 @@ function Test-MtExoMoeraMailActivity {
             return $null
         }
     }
+    $file = "$([System.IO.Path]::GetTempPath())\maester-EmailActivityUserDetail.csv"
```

Comment:
Nice tweak! One change, though: the GetTempPath method returns a path that includes a trailing path separator, so the '\\' in this line is unnecessary. (It could also cause problems in non-windows environments.)